README.md

```diff
@@ -1,2 +1,90 @@
-# coverter-go
-Convert AsyncAPI documents from older to newer versions.
+# AsyncAPI Converter
+
+Convert AsyncAPI documents from version 1.x to 2.0.0-rc1.
+
+## Installation
+
+`go get github.com/asyncapi/converter-go`
+
+## Usage
+
+### From CLI
+
```

Comment:
Add an introductory sentence to this section describing this way of using the AsyncAPI Converter, please.